Tasks/NuGetAuthenticateV0/main.ts

```diff
@@ -9,7 +9,6 @@ async function main(): Promise<void> {
     let forceReinstallCredentialProvider = null;
     try {
         tl.setResourcePath(path.join(__dirname, 'task.json'));
-        tl.setResourcePath(path.join(__dirname, 'node_modules/azure-pipelines-tasks-artifacts-common/module.json'));
```

Comment:
Could you please check if we already set it in artifacts-common package?

Reply:
Yes, sure! It's imported for every file which contains any localization strings usage.